### PR TITLE
buildscript translation support re-implementation

### PR DESCRIPTION
**Pull request by rbohl**
cc @thektan @sez11a
Kevin if you aren't familiar with recent events, the buildscript was reset back to the dawn of Liferay Learn. Translation support is one of the priorities to restore, so for the most part I've replayed your work. I think I have a working build right now and am ready to send it to bchan, but I am fishing for input from anyone familiar with what the requirements are before I pull the trigger. I'd love it if you guys could run the build and inspect the built site.
I'll send within an hour, so let me know if you see anything terrible here. Thanks guys!

### CODE REVIEW

**Comment by thektan** [comment]
Taking a look now.

**Comment by rbohl** [comment]
I'll hold off sending it for a bit since Kevin and JR are checking it out. I'll keep my eye on things here and on slack this evening (Eastern Time).

File: site/build_site.sh [hunk 1]
```diff
@@ -160,4 +176,4 @@ function upload_to_server {
 	echo upload_to_server
 }
 
-main "${@}"
\ No newline at end of file
+main "${@}"
```
Comment (thektan) [inline site/build_site.sh:179]: On macOS, I get the error
```
./build_site.sh: line 179: @: unbound variable
```

Previously we removed `"${@}"`
https://github.com/liferay/liferay-learn/commit/517397827645b6187bd2af8110f48b1e185d6e4a

Or another solution I've seen while googling around is adding a default.
```
main "${@:-}"
```

Comment (rbohl) [reply site/build_site.sh:179]: Good catch. That was one of the things that got reverted. After getting the production build case working with the language locales on Linux (that's for CI, and JR can build for production in his Windows Subsystem for Linux) with translations (this work I've done) we can begin tackling those other things (again :sweat_smile: ). Thanks, Kevin!

**Comment by rbohl** [comment]
Forwarded :rocket: : https://github.com/brianchandotcom/liferay-learn/pull/18

**Pull request closed without merging**